### ISSUE TITLE
RotationX in CoreShell

### ISSUE THREAD

**Issue by cykim66** [scgmlz/BornAgain#1212]
Recently, I tried to put  RotationX to CoreShell structure, but got exit error. 

Basically my particles are uniform cube shape core coated by a shell (close to cube shape).
I tilted (RotationX) CoreShell and got response that CoreShell does not have attribute of RotationX.
I tilted cube core and added tiled shell. This didn’t work neither. 

I ended up using "truncated sphere core-shell model " as an approximation and published result in open access journal, 
https://doi.org/10.3390/condmat5030054. 

In short, can CoreShell have RotationX attribute?

Thank you. 

Chang-Yong Kim

**Comment by jwuttke** [comment]
Dear Chang-Yong, thank you for this question.

`ParticleCoreShell` inherits from `IParticle`, hence has a method `setRotation`, hence is supposed to support `RotationX`. If it does not, then this is a bug. The more so if the program exits without proper error message.

Could you possibly supply us a script that demonstrates the unintended behavior?

Kind regards, Joachim

**Comment by cykim66** [comment]
Dear Joachim, 

Actually I communicated with you before and you suggested to submit "issue tracker".Attached is the code.Thank you.
Chang-Yong


    On Monday, February 15, 2021, 9:39:47 AM CST, Joachim Wuttke <notifications@github.com> wrote:  
 
 


Dear Chang-Yong, thank you for this question.

ParticleCoreShell inherits from IParticle, hence has a method setRotation, hence is supposed to support RotationX. If it does not, then this is a bug. The more so if the program exits without proper error message.

Could you possibly supply us a script that demonstrates the unintended behavior?

Kind regards, Joachim

**Comment by jwuttke** [comment]
Will be addressed as part of https://jugit.fz-juelich.de/mlz/bornagain/-/issues/10.
